### PR TITLE
Minor purge QoL update

### PR DESCRIPTION
**Pull request by sandlerino**
Added automatic deletion of the command message and info message of the purge aftermath if the purge is succesfully executed. Also added handling for negative numbers.

### CODE REVIEW

**Review by ArcasCZ — APPROVED** [review]
LGTM

File: po/sk.popie [hunk 1]
```diff
@@ -37,8 +37,8 @@ msgstr Akcia zrušená.
 msgid Number of messages to delete
 msgstr Počet správ na vymazanie.
 
-msgid Please use either a reply or provide a number of messages to delete.
-msgstr Prosím použi odpoveď na správu, alebo počet správ na zmazanie.
+msgid Please use either a reply or provide a positive number of messages to delete.
+msgstr Prosím použi odpoveď na správu, alebo kladný počet správ na zmazanie. Nie oboje.
```
Comment (jedla97) [inline po/sk.popie]: This translation seems to be wrong as there is added sentence.

Comment (sandlerino) [reply po/sk.popie]: My monkey brain was copypasting translations, should be fixed now.

**Review by BorekZnovustvoritel — APPROVED** [review]
Looks good to me.

**Pull request merged**